### ISSUE TITLE
Arithmetic in base `n`

### ISSUE THREAD

**Issue by siddhartha-gadgil** [siddhartha-gadgil/LTS2019#5]
* A number in base `n` is a vector/list with values in `Fin(n)`.
* Convert between these and natural numbers (for `n` positive).
* Define addition _with carry over_ etc in terms of such representations.
* Show that these operations coincide with those on natural numbers/integers.

**Comment by Chinmaya-Kausik** [comment]
Do we also consider subtraction?

**Comment by siddhartha-gadgil** [comment]
More importantly, have you proved that this coincides with natural number
addition and multiplication?

**Comment by Chinmaya-Kausik** [comment]
Not at all. I apologize if my statement about Arithmetic in base n being set was unclear - I meant all the groundwork is set and we can now start proving things.
